root/etc/s6-overlay/s6-rc.d/svc-apiping/run

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/with-contenv bash
+
+# Set defaults on first run
+if [ ! -f '/config/core.conf' ]; then
+  sleep 5
+  deluge-console --quiet --config=/config config --set download_location /downloads
```

Comment:
Does Deluge create core.conf by itself on first run?

Reply:
Yeah so it looks like we've been essentially operating on a race condition until now.

Deluged creates the config files but because the two services were starting at the same time the core.conf doesn't exist when the apiping service checks it, then it sleeps for 5 seconds to let deluged finish starting and then updates the core.conf that has been created.

Because I made the apiping and deluge-web depend on deluged to start, it now breaks.

Reply:
My biggest concern was to not force locations on existing locations

Reply:
It won't overwrite any existing configuration in any case, but there's definitely cleaner ways to handle this now that I've actually looked at what it's doing in detail.

Reply:
I wonder if it's as simple as changing $HOME to /config